### PR TITLE
Keyboard fixes, general code improvements

### PR DESCRIPTION
**Pull request by wrrath**
Fixed keyboard import issues and did some code cleanup.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -8,75 +9,62 @@
...
-        #Wiping after logo
-        pygame.display.set_caption('Player Selection')
-        screen.fill("black")
-        title()
-        # Setup player selection environment
-        pygame.draw.rect(screen, red, pygame.Rect(100, 50, X/2, Y-150))
-        pygame.draw.rect(screen, green, pygame.Rect((X/2), 50, X/2-100, Y-150))
         pygame.display.flip()
-        clock.tick(60)  # limits FPS to 60
+        clock.tick(600)  # limits FPS to 60
```
Comment (BatSmacker84) [inline main.py:60]: I would probably keep this capped at 60 for now

File: main.py [hunk 2]
```diff
@@ -8,75 +9,62 @@
...
         events()
+        print('fps is: ' + str(clock.get_fps()))
```
Comment (BatSmacker84) [inline main.py:62]: This actually managed to crash my desktop, so I would recommend removing it. All it does is fill the terminal with useless garbage anyways.

**Review by BatSmacker84 — CHANGES_REQUESTED** [review]
I'm having some issues with the window reloading over and over, but that could be a me issue.
I'll look at it on a more normal machine tomorrow and hopefully it is just a me issue.

File: main.py (hunk 1 above)
Comment (wrrath) [reply main.py:60]: Agreed, it was in there for some testing purposes

File: main.py (hunk 2 above)
Comment (wrrath) [reply main.py:62]: Agreed, again, just in for testing purposes

**Comment by wrrath** [comment]
> I'm having some issues with the window reloading over and over, but that could be a me issue. I'll look at it on a more normal machine tomorrow and hopefully it is just a me issue.

I tried to replicate on my machine.  The only thing in the code that would refresh the window is if there is a change between full screen and windowed mode.

**Comment by empotts** [comment]
Closed after talking to Russell

**Pull request closed without merging**